gems/sorbet/lib/fetch-rbis.rb

```diff
@@ -114,11 +128,12 @@ def self.main
 
     if vendor_paths.length > 0
       vendor_rbis_within_paths(vendor_paths)
+      remove_extra_rbis(vendor_paths)
```

Comment:
This one doesn't make sense to me. Can we not just remove the entire folder before re-generating, rather than having to selectively delete things after the fact? It looks like we already do this for generating `sorbet/rbi/gems` and `sorbet/rbi/hidden-definitions`:

https://github.com/sorbet/sorbet/blob/c404b8915d19147da3ef68de4fe7fde2e4caf48d/gems/sorbet/lib/gem-generator-tracepoint.rb#L42

Reply:
ah, yeah that would be much simpler. I'll change it to do that.